azure-templates/steps-pre-build.yml

```diff
@@ -5,6 +5,7 @@ parameters:
     type: string
   - name: quarterlyReleaseVersion
     type: string
+    default: 1999
```

Comment:
Is this just an obviously wrong default? Do we need to specify a default at all?

Reply:
This is an obviously wrong default that is there to prevent any existing pipelines from failing before they are updated to stop providing the quarterlyReleaseVersion

Reply:
For clarity, if no default is provided, then the parameter is required, which will cause the any yaml that doesn't specify this to fail.  Alternatively, if the parameter is removed, any yaml that specifies it will fail.  the default is a middle ground where both of these states will work, so we don't have all of the pipelines fail when this goes in for one reason or another.

Since all existing pipeline yaml do specify quarterlyReleaseVersion, but I want the PR for build-tools to test not using it, I have it as a default for now.